src/main/resources/calendar/air-bnb/airBnbCalendar-3.ics

```diff
@@ -0,0 +1,19 @@
+BEGIN:VCALENDAR
```

Comment:
do we need this file?

Reply:
Only for testing via Postman... I will delete calendar directory before the push